scripts/ci/verify-manifest-tools.py

```
@@ -54,88 +54,34 @@ def _parse_version(output: str, tool_name: str) -> str | None:
-    standard ``<binary> --version`` pattern.  Falls back to the install
-    block's ``bin`` field (or the tool name) plus ``--version``.
-
-    Args:
-        tool_name: Canonical tool name.
-        install: The ``install`` block from the manifest entry.
+    version_command = tool_entry.get("version_command")
+    if not isinstance(version_command, list) or not version_command:
+        raise ValueError(
+            f"tool {tool_name!r} requires a non-empty 'version_command' list, "
+            f"got {version_command!r}",
+        )
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)